src/bag3_digital/schematic/inv.py

```diff
@@ -54,6 +54,7 @@ def get_params_info(cls) -> Dict[str, str]:
             stack_n='number of transistors in a stack.',
             p_in_gate_numbers='a List indicating input number of the gate',
             n_in_gate_numbers='a List indicating input number of the gate',
+            vertical_out='True to draw vertical output',
```

Comment:
This needs a default parameter.

Reply:
This parameter name doesn't make sense. Schematic should not care about whether output is on vertical metal or not - that is a layout parameter. You probably meant to use 'separate_out', which is added in my PR #26.

Reply:
So do I just revert this change and wait for #26 to be merged?

Reply:
Change the variable name to `separate_out`. We'll deal with the merge later, since #26 is going into a different branch.

Reply:
agreed with Bob's suggestion, but be careful about a few details:
1. Sean probably intended separate_out = not(vertical_out), so the logic should be flipped.
2. It is in fact possible to have vertical_out = True and separate_out = True simultaneously (nout and pout both exported on vertical metal, just not shorted).
Easiest approach may be to just copy my implementation from #26.